addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1732,6 +1733,36 @@ QUnit.test('first unseen message should be directly preceded by the new message
     );
 });
 
+QUnit.test('focus should be on textarea on manual send [REQUIRE FOCUS]', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records = [{
+        channel_type: 'channel',
+        id: 20,
+        is_pinned: true,
+        name: "General",
+        uuid: 'channel20uuid',
+    }];
```

Comment:
Use `push` into `records` instead of assignation.

Also you don't need to define the parameters that are not necessary for the test (if I'm not wrong: channel_type doesn't matter, is_pinned is true by default, name is not used, and uuid is not used either).